docs/non-normative/groups/system/cpu-metrics-guidelines.md

```diff
@@ -0,0 +1,107 @@
+## Recommended vs Opt-In CPU Metrics
```

Comment:
This should be a level-1 header. For a fix, see:

- #2823 

/cc @lmolkova @trask